web/ci.sh

```diff
@@ -83,4 +89,108 @@ if builder_start_action validate-size; then
   ./src/tools/building/check-build-size.sh $FLAGS
 
   builder_finish_action success validate-size
+fi
+
+if builder_start_action prepare-s.keyman; then
+  # First phase: make sure the s.keyman.com repo is locally-available and up to date.
+  pushd "$S_KEYMAN_COM"
+
+  # For testing on a local development machine / a machine with the repo already loaded.
+  git checkout master
+  git pull
+  popd
+
+  # Second phase:  copy the artifacts over
+
+  # The main build products are expected to reside at the root of this folder.
+  BASE_PUBLISH_FOLDER="$S_KEYMAN_COM/kmw/engine/$BUILD_NUMBER"
+  mkdir "$BASE_PUBLISH_FOLDER"
+
+  cp -Rf build/app/web/release/* "$BASE_PUBLISH_FOLDER"
+  cp -Rf build/app/ui/release/* "$BASE_PUBLISH_FOLDER"
+
+  # Third phase: tweak the sourcemaps
+  # We can use an alt-mode of Web's sourcemap-root tool for this.
+  for sourcemap in "$BASE_PUBLISH_FOLDER/"*.map; do
+    node build/tools/building/sourcemap-root/index.mjs null "$sourcemap" --sourceRoot "https://s.keyman.com/kmw/engine/$BUILD_NUMBER/src"
```

Comment:
Can we safely assume here that this script is available?

Reply:
This is built as part of the main KMW build; unless the CI build calls are deliberately excluding it, it'll exist.